sbol3/object.py

```diff
@@ -3,7 +3,7 @@
 from collections import defaultdict
 from typing import Optional
 from urllib.parse import urlparse
-
+from typing import Dict, Callable
```

Comment:
Can this be cleaned up so the `typing` imports are all on one line, and there is a blank line between core Python imports and the local import (`from . import *`)?

Reply:
fixed